pulsar-client-cpp/python/schema_test.py

```diff
@@ -354,6 +354,32 @@ class Example(Record):
         self.assertEqual(r2.__class__.__name__, 'Example')
         self.assertEqual(r2, r)
 
+    def test_schema_version(self):
+        class Example(Record):
+            a = Integer()
+            b = Integer()
+
+        client = pulsar.Client(self.serviceUrl)
+        producer = client.create_producer(
+                        'my-avro-python-schema-version-topic',
+                        schema=AvroSchema(Example))
+
+        consumer = client.subscribe('my-avro-python-schema-version-topic', 'sub-1',
+                                    schema=AvroSchema(Example))
+        
+        r = Example(a=1, b=2)
+        producer.send(r)
+
+        msg = consumer.receive()
+
+        self.assertIsNotNone(msg.schema_version())
+
+        self.assertEqual(0, int.from_bytes(msg.schema_version().encode(), byteorder='big'))
```

Comment:
`int.from_bytes` is a Python3 method, but CI environment use Python2 to test. You need to use another method to convert bytes to int.

Reply:
Fixed as suggested. Thank you!